frontend/app/enrollment/components/selected-patient-view.tsx

```diff
@@ -0,0 +1,48 @@
+"use client"
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card'
+import React, { useEffect, useState } from 'react'
+import useEnrollmentStore from '@/lib/store/enrollment-store'
+import Image from 'next/image'
+import { getBsn } from '@/lib/fhirUtils'
+
+export default function SelectedPatientView() {
+
+    const { patient } = useEnrollmentStore()
+    const bsn = getBsn(patient)
+
+    return (
+        <Card className='bg-primary text-gray-100'>
+            <CardHeader>
+                <CardTitle>Patient Information</CardTitle>
+            </CardHeader>
+            <CardContent>
+                <div className="grid grid-cols-4 gap-4">
+                    <div className="col-span-1 bg-gray-100 p-6 rounded-lg shadow-md flex items-center justify-center">
+                        <Image
```

Comment:
we should read this from the Patient resource or leave it out?

Reply:
Sure, I'll use the photo attribute and a default placeholder

Reply:
https://linear.app/zorg-bij-jou/issue/INT-227/rework-frontend-patient-photo